### PR TITLE
Create list contacts UI

### PR DESCRIPTION
**Pull request by aagneyanNamboodiri**
Added the contact listing UI

### CODE REVIEW

File: app/javascript/src/components/Common/Sidebar/constants.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // @ts-ignore
```
Comment (yedhink) [inline app/javascript/src/components/Common/Sidebar/constants.js:1]: Go through the Slack thread.

File: app/javascript/src/components/Dashboard/Contacts/constants.jsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+import { MenuHorizontal } from "neetoicons";
+import { Typography, Dropdown, Avatar } from "neetoui";
+
+const MenuIcon = () => (
+  <Dropdown buttonStyle="text" icon={MenuHorizontal}>
+    <li>Edit</li>
+    <li>Delete</li>
+  </Dropdown>
+);
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/constants.jsx:11]: The component name doesn't resonate with the content. I think calling this component something like `ActionDropdown` would be better.

File: app/javascript/src/components/Dashboard/Notes/constants.js [hunk 3]
```diff
@@ -1,14 +1,7 @@
...
-  "Onboarding",
-  "User Flow",
-  "UX",
-  "Bugs",
-  "V2",
-];
+const CONTACTS = ["Oliver Smith", "Bilbo Baggins", "Frodo Baggins"];
+const TAGS = ["Getting started", "Onboarding", "User Flow", "UX", "Bugs", "V2"];
 
 export const CONTACT_OPTIONS = [
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/constants.js:6]: Create the options using the base entity. Here you already have `CONTACTS`. So you can create `CONTACT_OPTIONS` by iterating through the `CONTACTS`. Refer slack thread.

File: app/javascript/src/components/Dashboard/Notes/constants.js [hunk 4]
```diff
@@ -62,8 +55,8 @@ export const NOTES_FORM_INITIAL_FORM_VALUES = {
 export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
   title: yup.string().required("Title is required"),
   description: yup.string().required("Description is required"),
-  contactList: yup.string().oneOf(CONTACTS_LIST),
-  tagList: yup.string().oneOf(TAGS_LIST),
+  contactList: yup.string().oneOf(CONTACTS),
+  tagList: yup.string().oneOf(TAGS),
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/constants.js:59]: This validation doesn't seem right to me. Please go through it once again and see if there are any issues and get back to me.

**Review by yedhink — COMMENTED** [review]
@aagneyanNamboodiri

File: app/javascript/src/components/Dashboard/Notes/constants.js (hunk 4 above)
Comment (aagneyanNamboodiri) [reply app/javascript/src/components/Dashboard/Notes/constants.js:59]: I have fixed the validation schema.

File: app/javascript/src/components/Dashboard/Notes/constants.js (hunk 3 above)
Comment (aagneyanNamboodiri) [reply app/javascript/src/components/Dashboard/Notes/constants.js:6]: Fixed.

File: app/javascript/src/components/Common/Sidebar/constants.js (hunk 1 above)
Comment (aagneyanNamboodiri) [reply app/javascript/src/components/Common/Sidebar/constants.js:1]: Removed

File: app/javascript/src/components/Dashboard/Contacts/constants.jsx (hunk 2 above)
Comment (aagneyanNamboodiri) [reply app/javascript/src/components/Dashboard/Contacts/constants.jsx:11]: Yes. `ActionDropdown` suits the component.

**Pull request merged**